### DISCUSSION TITLE
Log files from ELOCs in the field

### DISCUSSION THREAD

**Discussion by EDsteve** [LIFsCode/ELOC-3.0#82] [General]
Was good to see you guys today. I like Fabi's idea to have a beer together one day :)

About the ELOCs in the jungle:
Alex is still without internet connection. So this is the only info i have so far:

From 4 devices which did not record any more after 2 days, he brought back one of the problematic ELOCs, kept the other three in the jungle and started recording again.

This is the log file from the retreived ELOC (ELOC_00051):
[elocConfig.config.zip](https://github.com/LIFsCode/ELOC-3.0/files/14433965/elocConfig.config.zip)

He tested ELOC_00051 at the base camp again and it stopped recording again. This is the log from the second attempt: 
[elocConfig.config (1).zip](https://github.com/LIFsCode/ELOC-3.0/files/14433967/elocConfig.config.1.zip)

The time stamps from the log files don't make sense to me (Do not unzip it. Just open the zip and check the time stamps).
Maybe that ELOC has an older firmware running?

ToDo:
Would it be a good idea to add the git-commit in the getConfig file? 

Will hopefully have more info and results within the next days.

**Comment by EDsteve** [comment]
### Alex also sent me the content of the recoring from the jungle:
![WhatsApp Image 2024-02-27 at 16 07 55](https://github.com/LIFsCode/ELOC-3.0/assets/25384076/2bb7929b-afde-429c-9d40-f07965938e41)

### and from the second recording fail:
![WhatsApp Image 2024-02-27 at 16 10 53](https://github.com/LIFsCode/ELOC-3.0/assets/25384076/78bee1e1-e947-46ce-8bbb-233316a15e87)

### This is the content of the eloc folder. 
![image](https://github.com/LIFsCode/ELOC-3.0/assets/25384076/b9145f43-aadc-4844-9878-0d308d8e87a6)

**Comment by LIFsCode** [comment]
@EDsteve I changed the init messages so the version and git commit will show up in the log file
[1f70807](https://github.com/LIFsCode/ELOC-3.0/commit/1f70807755fd76e3447ce7c6bf163666c5752f85)


In the log files (eloc.log1) you provided you could see that the ELOC rebooted:
```
[0;32mI (8739) Logging: 20:50:20 Tue, Feb 13 2024: start logging to sd card[0m
[0;32mI (8749) main: Creating Bluetooth  task...[0m
[0;32mI (8754) BluetoothServer: Enable BT with node ELOC_00051[0m
[0;32mI (8760) BTDM_INIT: BT controller compile version [8020d24][0m
[0;32mI (8768) system_api: Base MAC address is not set[0m
```

Also it can be seen that there was quite a lot of activity on the LIS3dh. Since there have been issues with the update (https://github.com/LIFsCode/ELOC-3.0/issues/83) are you sure that this is not  caused by the fixed bug https://github.com/LIFsCode/ELOC-3.0/issues/66

```
[0;33mW (1147259) I2SMEMSSampler: Audio sample clips occurred 816 times[0m
[0;33mW (1147339) I2SMEMSSampler: Audio sample clips occurred 397 times[0m
[0;33mW (1147420) I2SMEMSSampler: Audio sample clips occurred 42 times[0m
[0;32mI (1147447) BluetoothServer: Knock knock knocking on Heaveans Door...[0m
[0;32mI (1147459) BluetoothServer: LIS3DH double click
[0m
[0;32mI (1147463) BluetoothServer: Bat Voltage 3.343
[0m
[0;32mI (1147467) BluetoothServer:  two following are heap size, total and max alloc [0m
[0;32mI (1147473) BluetoothServer:      90648[0m
[0;33mW (1147485) I2SMEMSSampler: Audio sample clips occurred 631 times[0m
[0;32mI (1147488) BluetoothServer:      31744[0m
[0;32mI (1147495) BluetoothServer: Enable BT with node ELOC_00051[0m
[0;32mI (1147504) BTDM_INIT: BT controller compile version [8020d24][0m
[0;33mW (1148514) I2SMEMSSampler: Audio sample clips occurred 113 times[0m
[0;33mW (1148636) I2SMEMSSampler: Audio sample clips occurred 622 times[0m
[0;33mW (1148701) I2SMEMSSampler: Audio sample clips occurred 144 times[0m
[0;32mI (1148754) BluetoothServer: SerialBT is ready ------------------------------------------------------ [0m
[0;32mI (1148775) BluetoothServer: Knock knock knocking on Heaveans Door...[0m
[0;32mI (1148785) BluetoothServer: LIS3DH double click
[0m
[0;32mI (1148794) BluetoothServer: Bat Voltage 3.343
```

**Comment by EDsteve** [comment]
@LIFsCode 
I just saw in the database where all the getStatus and getConfig messages are being uploaded, that this ELOC is using Firmware version 0.1. Should have checked that earlier. Sorry about that.
So i guess this is one of the ELOCs where the update wasn't updated sucessfully. I will close this now. 

It's hard to query in the Firestore database right now. But Joseph is working on a website where we can easily search and filter through the ELOC metadata. Should be done in a few days, which should make things easier for us to see what is going on out there :)
